src/App.jsx

```diff
@@ -25,21 +26,11 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
+    <Person person={misha} />
 
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
+    <Person person={olya} />
+
+    <Person person={alex} />
```

Comment:
No need in empty lines between 1-line siblings.
```suggestion
    <Person person={olya} />
    <Person person={alex} />
```